advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: 'Using Commvault Backup & Recovery'
+description: 'Walkthroughs of multiple Commvault Backup & Recovery usage scenarios'
+---
+
+How to backups and restore from EDB Postgres Advanced Server using Commvault Backup & Recovery.
+
+## Using Commvault Backup & Recovery
+
+Commvault provides the two methods of taking the backup from EDB Postgres Advanced Server and restoring it.
+
+1. DumpBasedBackupSet Backup and Restore
+2. FSBasedBackupSet Backup and Restore
+
+Following examples shows how to use both methods of taking and restoring the backups.
+
+### DumpBasedBackupSet Backup and Restore
```

Comment:
Would be good to have this 
"DumpBasedBackupSet Backup and Restore"
and
"FSBasedBackupSet Backup and Restore"
as links on the right hand side so people could click them and go right to those section.  Ask Jasmer about this if you don't know how to enable it.